_maps/map_files/VoidRaptor/VoidRaptor.dmm

```diff
@@ -14285,7 +14285,6 @@
 /area/station/hallway/primary/central/fore)
 "emW" = (
 /obj/machinery/computer/cryopod{
-	dir = 8;
```

Comment:
![image](https://user-images.githubusercontent.com/8881105/190257984-1cb40857-9b69-43ea-8f5c-ae8f3914cb7e.png)
seems you've accidentally removed the dir on this which makes it at the wrong angle

Reply:
whoops, okay he's turned the right way round now.